code/__DEFINES/snpc.dm

```diff
@@ -14,10 +14,10 @@
 #define SNPC_FIGHTING 8
 #define SNPC_SPECIAL 16
 
```

Comment:
Due to namespace conflict (TRAIT_DUMB in specific), I decided to rename them all by adding AI_ to all the define